actions/testabilities/ability_spvwallet_application.go

```diff
@@ -0,0 +1,11 @@
+package testabilities
+
+import "testing"
+
+func New(t testing.TB) (given SPVWalletApplicationFixture, then SPVWalletApplicationAssertions) {
+	return Given(t), Then(t)
+}
```

Comment:
The `New` function seems to be unused.

Reply:
yes, in this certain example. Because we're reusing the same spv-wallet for each call, 
but it will be common to start spv-wallet in each test separately to not have polluted db,
so I don't want to remove it just to restore it soon.